res/controllers/Hercules_DJControl_Inpulse_300.midi.xml

```diff
@@ -248,14 +258,24 @@
 			<!--Q B-->
 			<control>
 				<group>[Channel2]</group>
-				<key>keylock</key>
+				<key>quantize</key>
```

Comment:
Confirmed. According to the manual Q = quantize on/off.

https://www.djuced.com/kb/hercules-djcontrol-inpulse-300-mapping-and-manual/

Shift + Q should be mapped to `beats_translate_curpos`

Reply:
I'd still like to keep the `key` mappings somewhere. Should I put it into "Toneplay" mode for pads?

Reply:
`beats_translate_curpos` mapping fixxed in f808227